CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Changes
 * [UI]: Fixed bug where a remote project in `UNSYNCHRONIZED` state would have the Sync buttons disabled on the Project Synchronization Settings page. (21.05.1)
 * [UI]: Added back `Users` option for managers in main navigation. (21.05.1)
 * [Documentation]: Added Swagger and OpenAPI to the IRIDA REST API documentation.
+* [UI]: Update of New Project page to use Ant Design.
```

Comment:
Change to `Update of Create New Project page to use Ant Design.`

Reply:
Updated in bafe5d04e62717eb92591a1af6f466c16b58b0c2